packages/app-accounts/src/index.tsx

```diff
@@ -68,6 +118,129 @@ class AccountsApp extends React.PureComponent<Props, State> {
   selectEdit = (): void => {
     this.setState({ action: 'edit' });
   }
+
+  selectRestore = (): void => {
```

Comment:
As per the comment about moving the rest of the functionality from the Editor above. What we have now is one monster component that contains all the logic for the whole app, instead of the different pieces in the seperate components where they belong. 

Basically we are creating an un maintainable monster here.

Reply:
Ok, I just felt I needed `accountAll` in index.tsx so I'd know whether any accounts existed in order to decide whether to render the Creator or Editor component, since we want it to automatically switch from Editor view to Creator view if the user clicks to forget all their accounts. So I'm deciding between whether to add the `accountAll` observable into index.tsx or to store a boolean state variable `isAccounts` in index.tsx that is updated by child components.

If the user restores an account in the Restorer component view I thought I'd need to set the restored keypair as the `current` keypair in the parent state of index.tsx so that the router would know to automatically switch to the Editor component view, with the `current` keypair shown in the selection box